README.md

```diff
@@ -2,23 +2,91 @@
 [![Build Status](https://api.travis-ci.org/kubernetes-incubator/cri-containerd.svg?style=flat-square)](https://travis-ci.org/kubernetes-incubator/cri-containerd)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-incubator/cri-containerd?style=flat-square)](https://goreportcard.com/report/github.com/kubernetes-incubator/cri-containerd)
 
-cri-containerd is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
+`cri-containerd` is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+`cri-containerd` is in alpha. This release is for use with Kubernetes 1.7. See
+the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Getting started
+### Binary Dependencies and Specifications
+The current release of cri-containerd has been tested with:
+* [containerd](https://github.com/containerd/containerd) v0.2.3-1098-g8ed1e24
+* [runc](https://github.com/opencontainers/runc) v1.0.0-rc3
+* [CNI](https://github.com/containernetworking/cni) v0.4.0
+
+As containerd and runc move to their respective general availability releases,
+we will do our best to rebase/retest `cri-containerd` with these releases on a
+weekly/monthly basis. Similarly, given that `cri-containerd` uses the Open
+Container Initiative (OCI) [image](https://github.com/opencontainers/image-spec)
+and [runtime](https://github.com/opencontainers/runtime-spec) specifications, we
+will also do our best to update `cri-containerd` to the latest releases of these
+specifications as appropriate.
+### Install Dependencies
+1. runc dependencies:
```

Comment:
nit: Install runc dependencies

A step should start with a verb. (Our document guys told me)